tests-wtr/dom-actions/dom-actions.click.ts

```diff
@@ -100,7 +95,6 @@ describe('click', () => {
     document.body.appendChild(shadowDiv)
     const shadow = shadowDiv.attachShadow({ mode: 'open' })
     const shadowButton = document.createElement('button')
-    shadowButton.innerText = '0'
```

Comment:
This was a remnant of the non-WTR way when increased number was printed into innerText